src/app/game/settings/page.tsx

```diff
@@ -0,0 +1,117 @@
+import NavigationFooter from "@/features/game/NavigationFooter";
+import Link from "next/link";
+import { Diamond,
+         Mail,
+         KeyRound,
+         LogOut,
+         Trash2,
+         Gamepad2,
+         Info,
+         FileText,
+         ChevronRight }
+         from 'lucide-react';
+
+export default function GamePage() {
+  const pageTitle = "設定";
+
+  return (
+    <div className="relative h-full">
+      <h1 className="fixed inset-x-0 top-0 bg-[#0094f4] p-4 pt-7 text-center text-3xl text-white">
+        {pageTitle}
+      </h1>
+
+    <div className="mx-auto justify-center pb-24 pt-28">
+        <div className="mx-auto mb-5 flex w-[90%] items-center">
+            <Diamond width={30} height={30} className="mr-5"/>
+
+            <span className="text-3xl">ユーザー設定</span>
+        </div>
+
+        <div className="mx-auto w-3/4">
+            <div className="mb-5 flex items-center">
+                <Mail width={30} height={30} className="mr-5"/>
+
+                <Link href={`/game/settings/account/change/email`}>
```

Comment:
ファイルパスにグレイブアクセント( \` )が用いられていますが、特に意図がなければ単純な二重引用符が良いと思いました！
```suggestion
                <Link href={"/game/settings/account/change/email"}>
```
もし、特に意図がなければ他の部分も修正お願いします！